### PR TITLE
build(flux): update flux to v0.174.0

### PR DESCRIPTION
**Pull request by scbrickley**

### CODE REVIEW

**Review by rockstar — APPROVED** [review]

**Comment by scbrickley** [comment]
@rockstar Seems like there are a few different failures in CI. Not sure what the contract is regarding the hand-off here. Does getting this green fall to me or you?

File: src/visitors/ast.rs [hunk 1]
```diff
@@ -227,18 +227,6 @@ impl<'a> walk::Visitor<'a> for SemanticTokenVisitor {
                     token_modifiers_bitset: 0,
                 });
             }
-            walk::Node::TestStmt(node) => {
```
Comment (rockstar) [inline src/visitors/ast.rs:230]: *slow clap*

File: integration/lsp.test.ts [hunk 2]
```diff
@@ -156,6 +156,6 @@ describe('module', () => {
         ast.body[0].init.value = '2';
         // Format into new source
         const src = format_from_js_file(ast);
-        expect(src).toBe('x = 2');
+        expect(src).toBe('x = 2\n');
```
Comment (rockstar) [inline integration/lsp.test.ts:159]: This all seems related to the formatter change. Just curious what the driving force behind this change was.

**Review by rockstar — APPROVED** [review]
One curiosity for my own edification, but please merge.

File: integration/lsp.test.ts (hunk 2 above)
Comment (scbrickley) [reply integration/lsp.test.ts:159]: I don't have the context for why it was necessary, but it was introduced here: https://github.com/influxdata/flux/pull/4905

Comment (onelson) [reply integration/lsp.test.ts:159]: iirc the deal with this was the formatter behaved differently when run on the CLI vs through the LSP with regards to the trailing newline. This brought the two back into alignment.

**Pull request merged**